docs/policy/collab-bearer-tokens.md

```diff
@@ -0,0 +1,134 @@
+
+If your collaboration centrally administers all services requiring bearer tokens and your users do not need to directly
+manage bearer tokens, consider running a simple token issuer.
+A simple token issuer consists of a public/private certificate keypair where the private key is used to issue tokens
+directly and the public certificate is made available through a web server.
+
+For example, the OSPool (née OSG VO) serves its public certificate through
+[GitHub pages](https://github.com/scitokens/osg-connect/) and uses the private key to sign tokens used for pilot job
+submission as well as automatically generating tokens to accompany user jobs so that they can access their private
+storage areas.
+
+### OAuth2/OpenID Connect ###
+
+If your collaboration distributes administrative responsibility or your users need to request and manage their own
+tokens,
+you should administer an OAuth2/OpenID Connect (OIDC) service (e.g., [INDIGO IAM](https://indigo-iam.github.io/v/v1.7.2/))
+or work with an existing OAuth2/OIDC provider (e.g., [CILogon](https://www.cilogon.org/subscribe)).
+
+For example, Fermilab uses CILogon as their OIDC provider combined with an
+[HTVault server](https://github.com/fermitools/htvault-config) used to streamline the OIDC process for users and
+integrate with their local Kerberos.
+
+Claims
+------
+
+Bearer tokens are self-describing credentials that enumerate their capabilities as "claims" and different token
+"profiles" enumerate common sets of claims.
+OSG sites support the following bearer token profiles:
+
+-   [SciTokens](https://scitokens.org/technical_docs/Claims)
+-   [WLCG tokens](https://github.com/WLCG-AuthZ-WG/common-jwt-profile/blob/master/profile.md)
+
+Claims are further described below with recommendations to ensure the greatest compatibility with OSG sites.
+
+### Scope ###
+
+The `scope` claim is a space-separated list of authorizations that should be granted to the bearer.
+Scopes utilized by OSG services include the following:
+
+| **Capability**   | **SciTokens scope** | **WLCG scope**                                 |
+|------------------|---------------------|------------------------------------------------|
+| HTCondor `READ`  | `condor:/READ`      | `compute.read`                                 |
+| HTCondor `WRITE` | `condor:/WRITE`     | `compute.modify compute.cancel compute.create` |
+| XRootD read      | `read:<PATH>`       | `storage.read:<PATH>`                          |
+| XRootD write     | `write:<PATH>`      | `storage.modify:<PATH>`                        |
+
+Replacing `<PATH>` with a path to the storage location that the bearer should be authorized to access.
+
+### Subject ###
+
+Subjects should be unique, stable identifiers that correspond to a user or service (e.g. pilot job submission).
+In other words, subjects combined with a token issuer can be used for suspending access for a given collaboration user,
+user-level accounting, monitoring, auditing, or tracing.
```

Comment:
```suggestion
In other words, subjects combined with a token issuer can be used for 
user-level accounting, monitoring, auditing, tracing, or suspension of access.
```
putting 'banning people' first made it a bit scary.

Reply:
It's probably the most "interesting" use case, though